test/others/test_schema.py

```diff
@@ -144,6 +201,25 @@ def test_label_to_dict(text_labels):
     assert l_new.answer.offsets_in_document[0].start == 1
 
 
+def test_equal_table_label(table_label_1, table_label_2):
+    assert table_label_1 == table_label_1
+    assert table_label_1 != table_label_2
```

Comment:
I get that we want to test equality but I think we should test it differently.
Let's create two identical answers and compare them, it's not valuable that valuable to compare something with itself.

We can split this into two separate tests actually, `test_labels_with_identical_fields_are_equal` and `test_labels_with_different_fields_are_not_equal`.

Reply:
I copied the format of the `test_equal_label` from above for text answers. Should we update those tests as well?

Reply:
Oh yeah! That would be great! <3